content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: 'Word choice'
+metaTitle: 'Prisma docs style guide: word choice'
+metaDescription: 'This section of the style guide provides guidelines on what terms and phrase constructions to use.'
+tocDepth: 3
+---
+
+## Avoid words like "easy" and "just"
+
+Avoid words like "easy", "just", "simple", and "basic". If users have a hard time completing the task that is supposedly "easy," then they will question their abilities. Consider using more specific descriptors. For example, when you say the phrase "deployment is easy," what do you really mean? Is it easy because it takes fewer steps than another option? If so, use the most specific descriptor possible, which in that case would be "this deployment method involves fewer steps than other options."
+
+For even more inclusive docs, avoid phrases that assume a reader’s experience or skill level, like "just deploy it and you’re done" or "for a refresher (referring to a completely different doc that someone may not have read)". Often, rephrasing results in stronger sentences that work in a wider range of contexts.
+
+## Avoid Latin terms
+
+These are problematic for non-first-language speakers, and some of them can be confusing even to native English speakers.
+
+For example:
+
+- et al (use "and others")
+- etc. (use "and so on", or list all of the cases)
+- i.e. (use "in other words")
+- e.g. (use "such as" or "for example")
+- via (use "with" or other equivalent, as in "Now you can start to send queries with the generated Prisma Client API")
+
+## Avoid gerunds ("ing" verb forms)
+
+Avoid gerunds (the "ing" form of verbs). For example, use "Get started" instead if "Getting started". This guideline applies to headings and body text.
+
+Examples:
+
+| Avoid                                           | Good                                      |
+| ----------------------------------------------- | ----------------------------------------- |
+| Test the certificate using a browser            | Test the certificate with a browser       |
+| Excluding fields                                | Exclude fields                            |
+| If you are using Node.js version 18 or later... | If you use Node.js version 18 or later... |
+| You can do this by rebuilding Prisma Client     | To do this, rebuild Prisma Client         |
+
+Note that nouns ending in "ing" are fine - for example "Tracing".
+
+## Avoid incomplete sentences before lists
+
+When you introduce a list, do not use an incomplete sentence.
+
+### Use
+
+You can do the following:
+
+- Item 1
+- Item 2
+
+### Avoid
+
+You can:
+
+- Item 1
+- Item 2
+
+## Referring to other parts of the docs
```

Comment:
At some point, we will want to add a section for writing about UIs... @zachtil  might want to do this part. For example, we will want to standardize on terms within a UI, such as "area", or "modal box", or "drop-down menu" or "left navigation pane"...

Reply:
Yep.

Reply:
I'll create an issue for this, and put it in the Style Guide epic

Reply:
Issue for this: https://github.com/prisma/docs/issues/3686